README.md

```diff
@@ -60,7 +60,27 @@ software. Some key features of this operator are:
 
 ### Custom resources
 
-// TODO: Add a summary of key `ContainerImageScan` (CIS) characteristics
+The Image Scanner operator does currently define a single user-facing Custom
+Resource Definition (CRD), [ContainerImageScan][CIS-CRD] (CIS), that represents the
+Kubernetes API for runtime image scanning of workload container images.
+See [stas_v1alpha1_containerimagescan.yaml][CIS-example] for a (simplified)
+example of a CIS resource.
+
+The CIS resource `.spec` specifies the container image to scan and some
+additional workload metadata, and the image scan result is added/updated
+in `.status` by the `ContainerImageScan` controller.
+
+CIS resources should not be edited by a standard users, as the `Workload`
+controller will create CIS'es from scheduled pods. And the standard Kubernetes
+garbage collector should delete obsolete resources when owning pods are gone.
+
+A user can influence the image scanning process by adding annotations to pods.
+The set of annotations is currently limited, but more might be added in the
+future:
+
+| Pod annotation key                         | Default value | Description                                                                                                         |
+|--------------------------------------------|:--------------|:--------------------------------------------------------------------------------------------------------------------|
+| `image-scanner.statnett.no/ignore-unfixed` | "false"       | If set to "true", the image-scanner will ignore any detected vulnerability that can't be fix by updating package(s) |
```

Comment:
```suggestion
| `image-scanner.statnett.no/ignore-unfixed` | `false`       | If set to `true`, the Image Scanner will ignore any detected vulnerability that can't be fix by updating package(s) |
```

Elsewhere in the description we have used *Image Scanner* consistently.

Reply:
I think `true` and `false` must be quote in annotations:
```suggestion
| `image-scanner.statnett.no/ignore-unfixed` |  `"false"`      | If set to `"true"`, the Image Scanner will ignore any detected vulnerability that can't be fix by updating package(s) |
```

Reply:
It is not the Image Scanner (operator) here, but the image-scanner.

Reply:
Good point, just to make it more clear:
```suggestion
| `image-scanner.statnett.no/ignore-unfixed` | `"false"`       | If set to `"true"`, the image-scanner controller will ignore any detected vulnerability that can't be fix by updating package(s) |
```

Reply:
But it is not the "image-scanner controller", but the image-scanner. 😉

Reply:
Ahh, as in Trivy?
```suggestion
| `image-scanner.statnett.no/ignore-unfixed` |  `"false"`       | If set to `"true"`, the image-scanner will ignore any detected vulnerability that can't be fix by updating package(s) |
```

Reply:
I have now fixed the quotes in quotes. Are you ok with image-scanner? Or image scanner? It is the scanner, and not the controller/operator I am trying to refer to here.

Reply:
> Ahh, as in Trivy?

Yes, indeed. Via the controller/operator, but...

Reply:
_image-scanner_ works for me 👍

Reply:
This was confusing to me. Since you are making a difference here, I think it could be slightly less subtle, e.g. *the image scanner engine*? Naming is hard!

Reply:
On second thought, I will change it to "Image Scanner". I on't really care, and prefer spending time on code. 😉